lib/create-testing-library-rule.ts

```diff
@@ -1,34 +1,35 @@
 import { ESLintUtils, TSESLint } from '@typescript-eslint/experimental-utils';
-import { getDocsUrl } from './utils';
+import { getDocsUrl, TestingLibraryRuleMeta } from './utils';
 import {
   detectTestingLibraryUtils,
   EnhancedRuleCreate,
 } from './detect-testing-library-utils';
 
-// These 2 types are copied from @typescript-eslint/experimental-utils
-type CreateRuleMetaDocs = Omit<TSESLint.RuleMetaDataDocs, 'url'>;
-type CreateRuleMeta<TMessageIds extends string> = {
-  docs: CreateRuleMetaDocs;
-} & Omit<TSESLint.RuleMetaData<TMessageIds>, 'docs'>;
-
 export function createTestingLibraryRule<
   TOptions extends readonly unknown[],
   TMessageIds extends string,
   TRuleListener extends TSESLint.RuleListener = TSESLint.RuleListener
 >(
   config: Readonly<{
     name: string;
-    meta: CreateRuleMeta<TMessageIds>;
+    meta: TestingLibraryRuleMeta<TMessageIds, TOptions>;
     defaultOptions: Readonly<TOptions>;
     create: EnhancedRuleCreate<TOptions, TMessageIds, TRuleListener>;
   }>
 ): TSESLint.RuleModule<TMessageIds, TOptions> {
-  const { create, ...remainingConfig } = config;
+  const { create, meta, ...remainingConfig } = config;
 
   return ESLintUtils.RuleCreator(getDocsUrl)({
     ...remainingConfig,
     create: detectTestingLibraryUtils<TOptions, TMessageIds, TRuleListener>(
       create
     ),
+    meta: {
+      ...meta,
+      docs: {
+        ...meta.docs,
+        recommended: false,
```

Comment:
Could you add a comment here indicating why we pass this always as `false` please?